### PR TITLE
added open_registrant_details_form in source_mixin.py

### PR DESCRIPTION
**Pull request by reichie020212**

### CODE REVIEW

**Review by nhatnm0612 — APPROVED** [review]

**Review by gonzalesedwin1123 — APPROVED** [review]

File: spp_change_request/models/mixins/source_mixin.py [hunk 1]
```diff
@@ -711,6 +711,28 @@ def action_attach_documents(self):
                     _("There are no directories defined for this change request.")
                 )
 
+    def open_registrant_details_form(self):
```
Comment (jeremi) [inline spp_change_request/models/mixins/source_mixin.py:714]: Add the documentation at the same time :D

**Review by jeremi — CHANGES_REQUESTED** [review]

**Review by gonzalesedwin1123 — APPROVED** [review]

**Comment by reichie020212** [comment]
@jeremi @gonzalesedwin1123 @nhatnm0612 @emjay0921 This PR is in conjunction with the PR on azure about the new CR Type. That particular PR is still on-hold since i'm still waiting for more info. This PR is still in draft state. Kindly do not merge this for now. Thanks.

File: spp_change_request/models/mixins/source_mixin.py (hunk 1 above)
Comment (reichie020212) [reply spp_change_request/models/mixins/source_mixin.py:714]: @jeremi noted on this.

**Comment by jeremi** [comment]
@reichie020212 if it is a draft make sure to mark it as draft. I moved it to draft.

**Pull request closed without merging**